les/api.go

```diff
@@ -104,12 +104,16 @@ func (api *PrivateLightServerAPI) clientInfo(c *clientInfo, id enode.ID) map[str
 		info["capacity"] = c.capacity
 		pb, nb := c.balanceTracker.getBalance(now)
 		info["pricing/balance"], info["pricing/negBalance"] = pb, nb
-		info["pricing/balanceMeta"] = c.balanceMetaInfo
+
+		cb := api.server.clientPool.ndb.getCurrencyBalance(id)
+		info["pricing/currency"] = cb.amount
 		info["priority"] = pb.base != 0
 	} else {
 		info["isConnected"] = false
-		pb := api.server.clientPool.ndb.getOrNewPB(id)
-		info["pricing/balance"], info["pricing/balanceMeta"] = pb.value, pb.meta
+		pb := api.server.clientPool.ndb.getOrNewBalance(id.Bytes(), false)
+
+		cb := api.server.clientPool.ndb.getCurrencyBalance(id)
+		info["pricing/balance"], info["pricing/currency"] = pb.value, cb.amount
```

Comment:
The actual `pricing/balance` should be `pb.value.value(logOffset)`. It's fixed by latter commit